shared/src/main/scala/FuncHelpers.scala

```diff
@@ -10,24 +10,51 @@ object FuncHelpers:
       fn.arity match
         case 1 =>
           val a = ctx.pop()
-          Monad.vectoriseNoFill { a =>
+          VList(ListHelpers.makeIterable(a).map { a =>
```

Comment:
Does this new implementation vectorise deeply, by the way? Or does it only do one level?

Reply:
It goes as deep as individual elements choose.

Reply:
Oh, so `v` does the same job as `M`? I thought it went deep, while `M` only went one level in.

Reply:
It's a sort of quick map thing that's good for when something doesn't vectorise by default

Reply:
Ah, I thought all this time that `v` meant you vectorised an element deeply.